src/main/java/com/fakng/fakngagrgtr/parser/stripe/StripeParser.java

```diff
@@ -0,0 +1,132 @@
+package com.fakng.fakngagrgtr.parser.stripe;
+
+import com.fakng.fakngagrgtr.parser.HtmlParser;
+import com.fakng.fakngagrgtr.parser.LocationProcessor;
+import com.fakng.fakngagrgtr.persistent.company.CompanyRepository;
+import com.fakng.fakngagrgtr.persistent.vacancy.Vacancy;
+import com.gargoylesoftware.htmlunit.WebClient;
+import com.gargoylesoftware.htmlunit.html.*;
+import lombok.SneakyThrows;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class StripeParser extends HtmlParser {
+
+    private final static String VACANCY_URL = "https://stripe.com/jobs/listing";
+    private final static String JOBS_PAGINATION_LIST_XPATH = "/html/body/div/section[4]/div/div[2]/div/div/nav/ul/li[7]/a";
+    private final static String VACANCY_TITLE_XPATH = "/html/body/section[1]/div/div[2]/div/div/section/header/h1";
+
+    public StripeParser(WebClient htmlWebClient,
+                        CompanyRepository companyRepository,
+                        LocationProcessor locationProcessor,
+                        @Value("${url.stripe}") String url) {
+        super(htmlWebClient, companyRepository, locationProcessor);
+        this.url = url;
+    }
+
+    @PostConstruct
+    public void init() {
+        initBase();
+    }
+
+    @Override
+    protected String getCompanyName() {
+        return "Stripe";
+    }
+    @Override
+    public List<Vacancy> getAllVacancies() throws IOException {
+        HtmlPage page = downloadPage(String.format(url, 0));
+        int pageCount = getPageCount(page);
+        List<Vacancy> vacancyList = new ArrayList<>();
+
+        for (int i = 0; i <= pageCount; i++) {
+            page = downloadPage(String.format(url, pageCount));
+            vacancyList.addAll(fillVacancy2(getLinksFromPage(page)));
+        }
+
+        return vacancyList;
+    }
+
+    private int getPageCount(HtmlPage page) {
+        return Integer.parseInt(page.getByXPath(JOBS_PAGINATION_LIST_XPATH).get(0).toString().replaceAll("\\D+", "")) / 100;
+    }
+
+    private DomNodeList<DomNode> getLinksFromPage(HtmlPage page) {
+        return page.querySelectorAll(".JobsListings__link");
```

Comment:
No way. This should be constant with a descriptive name. Moreover, you should use XPath, because it's more precise